packages/rs-sdk/src/platform/transition/vote.rs

```diff
@@ -79,8 +100,26 @@ impl<S: Signer> PutVote<S> for Vote {
         signer: &S,
         settings: Option<PutSettings>,
     ) -> Result<Vote, Error> {
+        let pub_key_hash = voting_public_key.public_key_hash()?;
+
+        let mut hasher = Sha256::new();
+        hasher.update(voter_pro_tx_hash.as_bytes());
+        hasher.update(pub_key_hash);
+        let voting_identity_id_hashed = hasher.finalize();
+        let voting_identity_id_bs58 = bs58::encode(voting_identity_id_hashed).into_string();
+        let voting_identity_id =
+            match Identifier::from_string(&voting_identity_id_bs58, Encoding::Base58) {
```

Comment:
Why not `Identifier::from_bytes()` ?

this will allow us to remove bs58 dependency